### PR TITLE
Hotfix 1.1.4

### PR DESCRIPTION
**Pull request by sven1103**
Fixes a misleading NPE

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <artifactId>java-openbis-dropboxes</artifactId>
-  <version>1.1.3-SNAPSHOT</version> <!-- <<QUBE_FORCE_BUMP>> -->
+  <version>1.1.4-SNAPSHOT</version> <!-- <<QUBE_FORCE_BUMP>> -->
```
Comment (wow-such-code) [inline pom.xml:7]: snapshot on main branch?

**Review by wow-such-code — CHANGES_REQUESTED** [review]
Version should be 1.1.4 (without snapshot) I think.

I am also concerned that getAllExceptions will not return the causes in a readable manner. Why does getCauses() not work?

File: pom.xml (hunk 1 above)
Comment (sven1103) [reply pom.xml:7]: the GA deployment script handles everything ;)

**Comment by sven1103** [comment]
> Version should be 1.1.4 (without snapshot) I think.
> 
> I am also concerned that getAllExceptions will not return the causes in a readable manner. Why does getCauses() not work?

The issue lies in the type conversion imo, the `getCauses()` returns a list of `DatasetValidationException`, but we convert it to `String messages`. That wont work.

File: src/main/groovy/life/qbic/registration/MainETL.groovy [hunk 2]
```diff
@@ -72,7 +72,7 @@ class MainETL extends AbstractJavaDataSetRegistrationDropboxV2 {
     }
 
     private static logDatasetValidations(DatasetValidationException exception) {
-        for(String message : exception.getCauses()) {
+        for(String message : exception.getAllExceptions()) {
```
Comment (wow-such-code) [inline src/main/groovy/life/qbic/registration/MainETL.groovy:75]: maybe we could use validationException.getAllMessages()? should work if this inherits from ValidationException

Comment (wow-such-code) [reply src/main/groovy/life/qbic/registration/MainETL.groovy:75]: turns out this is the wrapper method

Comment (wow-such-code) [reply src/main/groovy/life/qbic/registration/MainETL.groovy:75]: all good

**Review by wow-such-code — APPROVED** [review]

**Pull request merged**